src/components/appfooter.main.jsx

```diff
@@ -27,33 +28,33 @@ function AppFooterMain() {
           <div id="footer_container_column_1" className="col-xs-12 col-sm-12 col-md-4 col-lg-4 text-xs-center text-sm-center text-md-left text-lg-left">
             <div className="footerColumn">
               <h5 className="text-uppercase">
-                Vestri
+                {intl.get('vestri')}
```

Comment:
yeah, Vestri *is* the store name, so store-name would make sense

Reply:
I've used the pattern where 'store-name' would refer to 'Store Name' string so I think that 'vestri' key is consistent.

I've actually debated is Vestri should be localized at all since it is the name of the store, but decided to add it since all other strings are in there. Name of the store might be different in different locales after all.